bosh/manifest.yml

```diff
@@ -0,0 +1,73 @@
+---
+name: admin-ui
+
+releases:
+- name: admin-ui
+  version: latest
+
+update:
+  canaries: 1
+  canary_watch_time: 30000-600000
+  max_in_flight: 1
+  serial: true
+  update_watch_time: 5000-600000
+
+stemcells:
+- alias: default
+  name: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
+  version: latest
+
+instance_groups:
+- name: admin-ui
+  instances: 1
+  azs: [z1]
+  networks:
+  - name: default
+  vm_type: admin-ui
+  persistent_disk_type: admin-ui
+  stemcell: default
+  jobs:
+  - name: admin_ui
+    release: admin-ui
+    properties:
+      admin_ui:
+        cloud_controller_uri: ((admin_cloud_controller_uri))
```

Comment:
How about calling this `cloud_controller_uri`? This isn't a uri that's specific to the admin ui--it's just where the cloud controller api lives.